### PR TITLE
load llama v3 weights into fuji

### PR DESCRIPTION
**Pull request by sychen52**

### CODE REVIEW

File: axlearn/common/param_converter.py [hunk 1]
```diff
@@ -2331,3 +2333,102 @@ def parameters_from_t5x_encoder_decoder(
...
+    - (fuji_model_name="fuji-3B-v3", llama_model_name="Llama-3.2-3B")
+    - (fuji_model_name="fuji-8B-v3", llama_model_name="Llama-3.1-8B")
+    - (fuji_model_name="fuji-70B-v3", llama_model_name="Llama-3.1-70B")
+
+    Args:
+        llama: A Llama model with type LlamaForCausalLM.
+        state: The state of a fuji model.
+
+    Returns:
+        NestedTensor containing with the same structure as state,but the weights are from llama.
```
Comment (markblee) [inline axlearn/common/param_converter.py:2363]:
```suggestion
        NestedTensor containing the same structure as state, but the weights are from llama.
```

File: axlearn/experiments/text/gpt/param_converter_test.py [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+            atol = 2e-3
+        assert np.allclose(fuji_logits, llama_logits, atol=atol), (
+            f"{fuji_logits[0,0,:10]} != {llama_logits[0,0,:10]}, "
+            f"{np.abs(fuji_logits - llama_logits).max()}"
+        )
+
+        prng_key, grad_prng_key = jax.random.split(prng_key, 2)
+        fuji_grad = compute_fuji_grad(grad_prng_key, fuji, state, input_batch)
+        llama_grad = compute_llama_grad(llama, torch_ids, state)
+        assert_allclose_in_nested_dict(fuji_grad, llama_grad, atol * 1e-2)
```
Comment (markblee) [inline axlearn/experiments/text/gpt/param_converter_test.py:148]: Does `self.assertNestedAllClose` not work?

**Review by markblee — APPROVED** [review]

File: axlearn/common/param_converter.py (hunk 1 above)
Comment (sychen52) [reply axlearn/common/param_converter.py]: Done

File: axlearn/experiments/text/gpt/param_converter_test.py (hunk 2 above)
Comment (sychen52) [reply axlearn/experiments/text/gpt/param_converter_test.py]: Done

**Comment by sychen52** [comment]
How do I run the other two CI checks. It says on hold.

**Comment by sychen52** [comment]
I have to move all tests to high cpu, otherwise, it will fail ci

File: axlearn/experiments/text/gpt/param_converter_test.py [hunk 3]
```diff
@@ -0,0 +1,161 @@
...
+        dict(
+            fuji_model_name="fuji-8B-v3",
+            llama_model_name="Llama-3.1-8B",
+        ),
+        dict(
+            fuji_model_name="fuji-70B-v3",
+            llama_model_name="Llama-3.1-70B",
+        ),
+    )
+    @pytest.mark.high_cpu
```
Comment (ruomingp) [inline axlearn/experiments/text/gpt/param_converter_test.py:96]: Are high_cpu tests still run at every CI?

**Review by ruomingp — APPROVED** [review]

File: axlearn/experiments/text/gpt/param_converter_test.py (hunk 3 above)
Comment (sychen52) [reply axlearn/experiments/text/gpt/param_converter_test.py:96]: I guess not. I tested them manually.

**Pull request merged**